distributed/worker_memory.py

```diff
@@ -417,6 +417,7 @@ def memory_monitor(self, nanny: Nanny) -> None:
         except (ProcessLookupError, psutil.NoSuchProcess, psutil.AccessDenied):
             return  # pragma: nocover
 
+        assert self.memory_limit
```

Comment:
`self.memory_limit` should be set by this point as it's used on the next line in a division, so asserting it to make `mypy` happy that it's not `None`.